src/AprsISLibrary/TcpConnection.cs

```diff
@@ -0,0 +1,64 @@
+﻿ namespace AprsSharp.Connections.AprsIs
+{
+    using System;
+    using System.IO;
+    using System.Net.Sockets;
+
+    /// <summary>
+    /// Represents a TcpConnection.
+    /// </summary>
+    public sealed class TcpConnection : ITcpConnection, IDisposable
+    {
+        private readonly TcpClient tcpClient = new TcpClient();
+        private NetworkStream? stream;
+        private StreamWriter? writer;
+        private StreamReader? reader;
+
+        /// <inheritdoc/>
+        public void Connect(string server, int port)
+        {
+            tcpClient.Connect(@server, port);
```

Comment:
`server` here is prefixed with `@`, which I believe is unnecessary...

Prefixing a variable name with `@` is a way to [allow a C# keyword to be used as a variable](https://stackoverflow.com/questions/429529/what-does-the-symbol-before-a-variable-name-mean-in-c). Since "server" doesn't appear to be a [C# keyword](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/), I think we should be safe to remove it.

```suggestion
            tcpClient.Connect(server, port);
```